src/scene/rotation.cpp

```diff
@@ -36,6 +36,9 @@
 #include <ghoul/misc/templatefactory.h>
 
 namespace {
+    // A Rotation object describes a specific rotation for a scene graph node, which may
```

Comment:
```suggestion
    // A `Rotation` object describes a specific rotation for a scene graph node, which may
```
is it an object? 🤔

Reply:
To the user it feels like it is, even though it is actually an abstract base class.  But that we can describe in the header file documentation.